diff -up kea-1.1.0-beta/ext/coroutine/coroutine.h.coroutinepgsql kea-1.1.0-beta/ext/coroutine/coroutine.h
--- kea-1.1.0-beta/ext/coroutine/coroutine.h.coroutinepgsql	2016-08-28 20:36:34.000000000 +0200
+++ kea-1.1.0-beta/ext/coroutine/coroutine.h	2016-09-01 09:59:16.767902664 +0200
@@ -108,7 +108,7 @@ private:
   for (_coro_value = __LINE__;;) \
     if (_coro_value == 0) \
     { \
-      case __LINE__: ; \
+/*      case __LINE__: ;*/ \
       break; \
     } \
     else \
diff -up kea-1.1.0-beta/src/lib/dhcpsrv/pgsql_host_data_source.cc.coroutinepgsql kea-1.1.0-beta/src/lib/dhcpsrv/pgsql_host_data_source.cc
--- kea-1.1.0-beta/src/lib/dhcpsrv/pgsql_host_data_source.cc.coroutinepgsql	2016-08-28 20:36:34.000000000 +0200
+++ kea-1.1.0-beta/src/lib/dhcpsrv/pgsql_host_data_source.cc	2016-09-02 19:48:37.210354247 +0200
@@ -1698,7 +1698,7 @@ std::pair<uint32_t, uint32_t> PgSqlHostD
     uint32_t minor;
     PgSqlExchange::getColumnValue(r, 0, 0, minor);
 
-    return (std::make_pair<uint32_t, uint32_t>(version, minor));
+    return (make_pair(version, minor));
 }
 
 void
diff -up kea-1.1.0-beta/src/lib/dhcpsrv/pgsql_lease_mgr.cc.coroutinepgsql kea-1.1.0-beta/src/lib/dhcpsrv/pgsql_lease_mgr.cc
--- kea-1.1.0-beta/src/lib/dhcpsrv/pgsql_lease_mgr.cc.coroutinepgsql	2016-08-28 20:36:34.000000000 +0200
+++ kea-1.1.0-beta/src/lib/dhcpsrv/pgsql_lease_mgr.cc	2016-09-01 09:59:16.767902664 +0200
@@ -1391,7 +1391,7 @@ PgSqlLeaseMgr::getVersion() const {
     tmp.str(PQgetvalue(r, 0, 1));
     tmp >> minor;
 
-    return make_pair<uint32_t, uint32_t>(version, minor);
+    return (make_pair(version, minor));
 }
 
 void
