diff -up ./ext/coroutine/coroutine.h.coroutinepgsql ./ext/coroutine/coroutine.h
--- ./ext/coroutine/coroutine.h.coroutinepgsql	2015-12-28 13:18:05.000000000 +0100
+++ ./ext/coroutine/coroutine.h	2016-03-15 14:58:26.344825900 +0100
@@ -108,7 +108,7 @@ private:
   for (_coro_value = __LINE__;;) \
     if (_coro_value == 0) \
     { \
-      case __LINE__: ; \
+/*      case __LINE__: ;*/ \
       break; \
     } \
     else \
diff -up ./src/lib/dhcpsrv/pgsql_lease_mgr.cc.coroutinepgsql ./src/lib/dhcpsrv/pgsql_lease_mgr.cc
--- ./src/lib/dhcpsrv/pgsql_lease_mgr.cc.coroutinepgsql	2015-12-28 13:18:45.000000000 +0100
+++ ./src/lib/dhcpsrv/pgsql_lease_mgr.cc	2016-03-15 13:59:06.837391677 +0100
@@ -1690,7 +1690,7 @@ PgSqlLeaseMgr::getVersion() const {
 
     PQclear(r);
 
-    return make_pair<uint32_t, uint32_t>(version, minor);
+    return (make_pair(version, minor));
 }
 
 void
