diff --git a/src/hooks/dhcp/high_availability/Makefile.am b/src/hooks/dhcp/high_availability/Makefile.am
index 26d782323..f0deb18e8 100644
--- a/src/hooks/dhcp/high_availability/Makefile.am
+++ b/src/hooks/dhcp/high_availability/Makefile.am
@@ -43,8 +43,8 @@ nodist_libha_la_SOURCES = ha_messages.cc ha_messages.h
 libha_la_CXXFLAGS = $(AM_CXXFLAGS)
 libha_la_CPPFLAGS = $(AM_CPPFLAGS)
 
-# install the shared object into $(libdir)/hooks
-lib_hooksdir = $(libdir)/hooks
+# install the shared object into $(libdir)/kea/hooks
+lib_hooksdir = $(libdir)/kea/hooks
 lib_hooks_LTLIBRARIES = libdhcp_ha.la
 
 libdhcp_ha_la_SOURCES  =
